docs/area-owners.md

```diff
@@ -65,7 +65,7 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
 | area-System.Collections                        | @ericstj      | @eiriktsarpalis @layomia                            | Excluded:<ul><li>System.Array -> System.Runtime</li></ul> Consultants: @steveharter @GrabYourPitchForks                                                                                                                                                                                   |
 | area-System.ComponentModel                     | @ericstj      | @maryamariyan @michaelgsharp @safern @tarekgh       |                                                                                                                                                                                                                                                                                           |
 | area-System.ComponentModel.Composition         | @jeffhandley  | @buyaa-n @joperezr @krwq                            | Consultants: @steveharter                                                                                                                                                                                                                                                                 |
-| area-System.ComponentModel.DataAnnotations     | @ajcvickers   | @lajones @ajcvickers                                |                                                                                                                                                                                                                                                                                           |
+| area-System.ComponentModel.DataAnnotations     | @ajcvickers   | @lajones @ajcvickers                                | Included:<ul><li>System.ComponentModel.Annotations</li></ul>                                                                                                                                                                                                                             |
```

Comment:
It might be worth mentioning that these are actually synonymous, rather than Included.  I'm fine either way.  This has always been a confusing assembly/package name.  My understanding of the history was that folks wanted a different assembly name than desktop to enable OOBing, but that was never done.  This has always been 100% type-forwards from System.ComponentModel.Annotations.dll to System.ComponentModel.DataAnnotations.dll

Reply:
Thanks for the details. I think for the purpose of this doc, the term "Included" is fine. Technical details shouldn't be captured here anyway.